food-bank-app/src/hooks/useClient/httpClient.ts

```diff
@@ -0,0 +1,38 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios'
+
+declare module 'axios' {
+  interface AxiosResponse<T = any> extends Promise<T> {}
+}
```

Comment:
Thanks to this, we can instantly can instantly cast the responses we receive.

E.g.
```javascript
this.instance.get<User>('/contributors/me')
```